lib/src/builder_dio.dart

```diff
@@ -18,11 +18,11 @@ Options buildServiceCacheOptions(
 
 /// build a normal cache options
 Options buildCacheOptions(Duration maxAge,
```

Comment:
Could this be ```Duration? maxAge``` instead?
The documentation said 
```
maxAge: set the cache time. If the value is null or not setted, it will try to get maxAge and maxStale from response headers.
```
It doesn't match the code anymore, and the code below still checking ```if (null != maxAge)` { ... }``
```

Reply:
Sure, that sounds right. This PR is already merged though so you can create a new PR maybe, which makes the parameter nullable?